Pod/ILGClasses/ILGClasses.m

```diff
@@ -66,9 +66,20 @@ + (NSSet *)subclassesOfClass:(Class)superclass
 
 + (NSSet *)classesConformingToProtocol:(Protocol *)protocol
 {
-    return [self classesPassingTest:^BOOL(Class class) {
+    NSSet *superclassesConforming = [self classesPassingTest:^BOOL(Class class) {
         return class_conformsToProtocol(class, protocol);
     }];
+    
+    NSMutableSet *subclasses = [NSMutableSet set];
+    
+    for (Class superclass in superclassesConforming) {
+        NSSet *subclassesOfSuper = [self subclassesOfClass:superclass];
+        [subclasses addObjectsFromArray:[subclassesOfSuper allObjects]];
+    }
+    
+    NSSet *allConforming = [superclassesConforming setByAddingObjectsFromSet:subclasses];
+    
+    return allConforming;
```

Comment:
My offhand thought was that we build up the collection of classes that pass the original test, then we do a single `classesPassingTest:` run where the test is like the one used by `subclassesOfClass:` but instead of checking for a single class, it checks for membership in the built-up collection.  It's something like O(n*m) where n is the total number of classes and m is the count of classes from the original test, which should be small, so better than O(n<sup>2</sup>).

It maybe could be a little better by dynamically adding any passing class to the list of known passing classes, but I'm not really sure if that'll be better or worse.